### PR TITLE
Fix issue with the wheel not including .sh files for merlin examples

### PR DESCRIPTION
**Pull request by bgunnar5**
The .sh files in the merlin examples weren't being added to the wheel so I modified MANIFEST.in to include them

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,6 +1,6 @@
 recursive-include merlin/data *.yaml *.py
 recursive-include merlin/server *.yaml *.py
-recursive-include merlin/examples *.py *.yaml *.c *.json *.sbatch *.bsub *.txt *.temp
+recursive-include merlin/examples *.py *.yaml *.c *.json *.sbatch *.bsub *.txt *.temp *.sh
```
Comment (koning) [inline MANIFEST.in:3]: Maybe this should just be * (untested)

**Comment by ryannova** [comment]
Are there other changes we want to include with version 1.9.2? I recommend we consolidate some of the changes/ fixes before we issue a new version.

**Comment by koning** [comment]
I've added a feature/flux_cluster branch but it needs a little more testing

File: MANIFEST.in (hunk 1 above)
Comment (bgunnar5) [reply MANIFEST.in:3]: I just did a quick glance through of the example workflow folders and I think you're right. I'm going to give it a shot before this gets merged.

Comment (bgunnar5) [reply MANIFEST.in]: This change worked. Just pushed it.

**Review by ryannova — APPROVED** [review]
LGTM

**Review by lucpeterson — APPROVED** [review]

**Comment by koning** [comment]
I think I found another bug. I'm running an openfoam singularity workflow and am using $(MERLIN_SAMPLE_PATH) but I see .../openfoam_wf_singularity/openfoam_wf_output/openfoam_wf_singularity_20221222-135351/sim_runs/00/00/ instead of the actual dir .../openfoam_wf_singularity/openfoam_wf_output/openfoam_wf_singularity_20221222-135351/sim_runs/00/. The sample dir in the var is /00/00 instead of /00/

**Comment by koning** [comment]
> I think I found another bug. I'm running an openfoam singularity workflow and am using $(MERLIN_SAMPLE_PATH) but I see .../openfoam_wf_singularity/openfoam_wf_output/openfoam_wf_singularity_20221222-135351/sim_runs/00/00/ instead of the actual dir .../openfoam_wf_singularity/openfoam_wf_output/openfoam_wf_singularity_20221222-135351/sim_runs/00/. The sample dir in the var is /00/00 instead of /00/

THis happend when the number o samples was 3, when it was set to 100, it gives the correct dir.

**Comment by lucpeterson** [comment]
i am unable to reproduce

**Comment by koning** [comment]
I gave you the workflow with the MERLIN_SAMPLE_PATH commented out, un-comment that CBIND to run with the sample path , this is with 1.9.1 develop in a dev venv. I'll recreate it to check.

**Comment by koning** [comment]
A reinstall seems to have fixed the issue. Sorry for the noise.

**Review by lucpeterson — APPROVED** [review]

**Pull request merged**